app/views/categories/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1><%= @category.name %></h1>
```

Comment:
Adds show view - very similar to items index, we should probably make a partial.